File: ios/Classes/FlutterInstallAppPlugin.m

#import "FlutterInstallAppPlugin.h"

#if __has_include(<flutter_install_app_plugin/flutter_install_app_plugin-Swift.h>)
#import <flutter_install_app_plugin/flutter_install_app_plugin-Swift.h>
#else
#import "flutter_install_app_plugin-Swift.h"
#endif

@implementation FlutterInstallAppPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  [SwiftFlutterInstallAppPlugin registerWithRegistrar:registrar];
}
@end

File: ios/Classes/FlutterInstallAppPlugin.swift
import Flutter
import UIKit

public class FlutterInstallAppPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let channel = FlutterMethodChannel(name: "flutter_install_app_plugin", binaryMessenger: registrar.messenger())
    let instance = FlutterInstallAppPlugin()
    registrar.addMethodCallDelegate(instance, channel: channel)
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    switch call.method {
    case "getPlatformVersion":
      result("iOS " + UIDevice.current.systemVersion)
    default:
      result(FlutterMethodNotImplemented)
    }
  }
}
